### PR TITLE
feat: add lightbox + landing for case-study

### PR DESCRIPTION
**Pull request by Gowiem**
## Release Notes

- **New Features**
	- Introduced a new case study detailing how Masterpoint helped Power Digital Marketing reduce infrastructure costs by 10x.
	- Added a lightbox feature for enhanced user interaction with case studies.
	- Launched a new landing page layout with responsive design elements and video embedding.
	- Enhanced styling for the landing banner and close button for improved visual presentation.

- **Documentation**
	- Added a thank-you page to express gratitude for case study downloads, including contact information for inquiries.

### CODE REVIEW

File: content/landing/power-digital-case-study.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+url: power-digital-case-study
+banner_button: /download/power-digital-case-study.pdf
+banner_button_text: Download Our PDF Case Study
+---
+
+[Power Digital Marketing](https://powerdigital.com), a rapidly growing digital marketing agency, faced scalability challenges with their existing Terraform Cloud infrastructure. With a complex setup supporting numerous clients and a growing number of resources, they needed a far more stable, cost-effective solution. Masterpoint stepped in, leveraging their deep Terraform expertise to migrate Power Digital from Terraform Cloud to Spacelift, a more flexible and cost-effective platform. Through careful planning and execution, Masterpoint successfully migrated over 43,000 resources, optimized the infrastructure, and implemented automation to support Power Digital Marketing’s future growth. 
+
+> The result was a **10x reduction** in infrastructure costs, **significantly faster** plan and apply cycles, and a **more resilient**, scalable infrastructure that can easily onboard new clients
+
+<a href="/download/power-digital-case-study.pdf" class="button">Download Our PDF Case Study &rsaquo;</a>
```
[the remark this answers is not in the document]
Comment (Gowiem) [reply content/landing/power-digital-case-study.md:22]: We don't want to do this. Please don't suggest this in the future.

File: content/landing/power-digital-case-study.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  priority: 0
+id: landing
+url: power-digital-case-study
+banner_button: /download/power-digital-case-study.pdf
+banner_button_text: Download Our PDF Case Study
+---
+
+[Power Digital Marketing](https://powerdigital.com), a rapidly growing digital marketing agency, faced scalability challenges with their existing Terraform Cloud infrastructure. With a complex setup supporting numerous clients and a growing number of resources, they needed a far more stable, cost-effective solution. Masterpoint stepped in, leveraging their deep Terraform expertise to migrate Power Digital from Terraform Cloud to Spacelift, a more flexible and cost-effective platform. Through careful planning and execution, Masterpoint successfully migrated over 43,000 resources, optimized the infrastructure, and implemented automation to support Power Digital Marketing’s future growth.
+
+> The result was a **10x reduction** in infrastructure automation costs, **significantly faster** plan and apply cycles, and a **more resilient**, scalable infrastructure that can easily onboard new clients
```
Comment (oycyc) [inline content/landing/power-digital-case-study.md:19]: Did someone say this or is this just a summary?

Comment (oycyc) [inline content/landing/power-digital-case-study.md:19]:
```suggestion
> The result was a **10x reduction** in infrastructure automation costs, **significantly faster** plan and apply cycles, and a **more resilient**, scalable infrastructure that can easily onboard new clients.
```

**Review by oycyc — CHANGES_REQUESTED** [review]

File: content/landing/power-digital-case-study.md (hunk 2 above)
Comment (oycyc) [reply content/landing/power-digital-case-study.md:19]: Nevermind, I thought it was intended to be like a quote.

**Review by oycyc — APPROVED** [review]

**Pull request merged**